docs/introduction-to-smart-contracts.rst

```diff
@@ -504,10 +504,10 @@ operations, loops should be preferred over recursive calls. Furthermore,
 only 63/64th of the gas can be forwarded in a message call, which causes a
 depth limit of a little less than 1000 in practice.
 
-.. index:: delegatecall, callcode, library
```

Comment:
Did you check whether some other docs are using this as ref/anchor? If there are, it might be that those sites require changes too.

Reply:
I have done `make clean && make html` and confirmed that warnings do not appear :)
Are there any patterns that this does not detect?

Reply:
Sounds good. Weirdly enough, none of these seem to be used as references.